ui/src/shared/utils/preventNegativeZero.test.ts

```diff
@@ -0,0 +1,43 @@
+import {preventNegativeZero} from 'src/shared/utils/preventNegativeZero'
+
+describe('preventNegativeZero', () => {
+  it('should not alter non-zero numbers', () => {
```

Comment:
nice test! was going to ask why you can't just use `Math.abs` but this one answers it nicely.